### PR TITLE
[PR CLONE]Synths

### PR DESCRIPTION
**Pull request by BobJogaIsForkinBanned**
## About The Pull Request

Basically balances synths so they can be a roundstart race, in a way. Synths are kind of funky so i'll have to do some weird fucking shit to make this work.
Basically, what i did is make it a fucking quirk - This may seem weird, but it is way easier for me since the synth species inherits shit from another species on being gained.
This also helps in one way, as you'll be able to tell who is a synth with a health analyzer by the quirk - but not just by normally examining.

## Why It's Good For The Game

I know for a fact there is some rando who roleplays as a synth (angel rainwater i think), so there is at least some demand for it
its synths they're the funny fallout 4 race or whatever
more roleplay opportunities yada yada
basically expensive IPCs i guess(???)

## Changelog
:cl:
add: Added a synth neutral quirk... which turns you into a synth once you spawn. Upsides and downsides are detailed in the desc. Fallout jokes yada yada yeah.
balance: Synths have been rebalanced for the synth quirk. Now they can be dismembered and their limbs can be disabled, and they breathe.
refactor: Synth species datum has been changed a bit so custom limb icons are taken into account etc.
/:cl:

### CODE REVIEW

**Comment by Strazyplus** [comment]
So this gives you a quirk that makes it so you cant get viruses and can only be repaired like a borg or with synthflesh, but still need to breathe air? 

why air still there?

**Comment by BobJogaIsForkinBanned** [comment]
> So this gives you a quirk that makes it so you cant get viruses and can only be repaired like a borg or with synthflesh, but still need to breathe air?
> 
> why air still there?
> 
> eh hell with it, I love it anyway. whats this about damage below 76 removes flesh. what do the sprites change to show your robotic self like a terminator? lol that'd be sick.

The need to breathe is purely for balance, otherwise powergamers will use it just for that.

**Comment by Akrilla** [comment]
probably @Useroth call if we're wanting this

**Comment by BobJogaIsForkinBanned** [comment]
> So this gives you a quirk that makes it so you cant get viruses and can only be repaired like a borg or with synthflesh, but still need to breathe air?
> 
> why air still there?
> 
> eh hell with it, I love it anyway. whats this about damage below 76 removes flesh. what do the sprites change to show your robotic self like a terminator? lol that'd be sick.

Also i tested it, their disguise doesn't fail in practice but injuries show the IPC overlay instead of the human one.

**Comment by CinderWC** [comment]
Merge this already i want to finally make a synth char for this @Useroth

**Comment by CinderWC** [comment]
DNM for now

**Comment by CinderWC** [comment]
**THIS IS READY TO TESTMERGE NOW**

@Useroth

**Comment by Commandersand** [comment]
TEST MERGE TEST MERGE

**Comment by Akrilla** [comment]
oh god oh fuck
